content/en/about/contributing/shortcodes/_index.md

```diff
@@ -144,102 +142,107 @@ in multiple programming languages, you can use a tabbed pane for code snippets:
 
 Will render as:
 
-{{< tabpane >}}
-{{< tab header="C++" lang="C++">}}
-#include <iostream>
-int main()
-{
-  std::cout << "Hello World!" << std::endl;
-}
-{{< /tab >}}
-{{< tab header="Java" lang="Java">}}
-class HelloWorld {
-  static public void main( String args[] ) {
-    System.out.println( "Hello World!" );
-  }
-}
-{{< /tab >}}
-{{< tab header="Kotlin" lang="Kotlin">}}
-fun main(args : Array<String>) {
-    println("Hello, world!")
-}
-{{< /tab >}}
-{{< tab header="Go" lang="Go">}}
-package main
-import "fmt"
-func main() {
-  fmt.Printf("Hello World!\n")
-}
-{{< /tab >}}
+{{< tabpane >}} {{< tab header="C++" lang="C++">}} #include <iostream> int
+main() { std::cout << "Hello World!" << std::endl; } {{< /tab >}}
+{{< tab header="Java" lang="Java">}} class HelloWorld { static public void main(
+String args[] ) { System.out.println( "Hello World!" ); } } {{< /tab >}}
+{{< tab header="Kotlin" lang="Kotlin">}} fun main(args : Array<String>) {
+println("Hello, world!") } {{< /tab >}} {{< tab header="Go" lang="Go">}} package
+main import "fmt" func main() { fmt.Printf("Hello World!\n") } {{< /tab >}}
 {{< /tabpane >}}
```

Comment:
I strongly disagree on this. This is unreadable, unmanageable, and a nightmare to fix. You have no idea how long it took to get these examples to render "almost" correctly (working around the render problems of nested code blocks).

Reply:
Not to mention that it renders wrong.

Reply:
Sure, this should be fixed now: https://docs.egi.eu/documentation/416/about/contributing/shortcodes/#code-in-multiple-languages